src/bgw/job.c

```diff
@@ -1176,8 +1177,15 @@ ts_bgw_job_entrypoint(PG_FUNCTION_ARGS)
 		zero_guc("max_parallel_workers");
 		zero_guc("max_parallel_maintenance_workers");
 
+		ts_begin_tss_store_callback();
+
 		res = ts_bgw_job_execute(job);
 
+		char *stmt = psprintf("CALL %s.%s($1, $2)",
+							  NameStr(job->fd.proc_schema),
+							  NameStr(job->fd.proc_name));
+		ts_end_tss_store_callback(stmt, -1, (int) strlen(stmt), 0, 0);
+
```

Comment:
Jobs can either be functions or procedures.
Not passing at least the job id limits visibility into what is happening

Reply:
> Jobs can either be functions or procedures. 

Do you think we need to use `SELECT` in the statement when it is a function?

> Not passing at least the job id limits visibility into what is happening

Didn't get your point here. Aren't queries we send to ts_stat_statement normalized??

Reply:
Found something interesting, pg_stat_statements don't normalize CALL statements:
```sql
fabrizio=# select userid, dbid, queryid, query from pg_stat_statements where query ~ '^CALL';
 userid | dbid  |       queryid        |                                                 query                                                 
--------+-------+----------------------+-------------------------------------------------------------------------------------------------------
     10 | 16384 | -6911009442551476100 | CALL _timescaledb_functions.policy_refresh_continuous_aggregate(2, '{"mat_hypertable_id": 2}'::jsonb)
     10 | 16384 |  -479759584010016119 | CALL _timescaledb_functions.policy_refresh_continuous_aggregate(2, NULL)
(2 rows)
```

Reply:
I think it will in PG17:
https://github.com/postgres/postgres/commit/11c34b342bd77a292b625c32b70b44c02db8f3ed

Reply:
By the way, to be clear, `ts_stat_statements` should get the arguments unnormalized.